distributed/stealing.py

```diff
@@ -417,4 +425,62 @@ def story(self, *keys):
         return out
 
 
+def _can_steal(ws, what, from_):
+    """Determine whether worker ``ws`` can steal task ``what`` from worker
+    ``from_``.
+    """
+    if what.loose_restrictions:
+        logger.debug("Task {} has loose restrictions".format(what.key))
```

Comment:
Typically logging uses old style string formatting, and the values are passed to `logger.debug`, .e.g


```
logger.debug("Task %s has loose restrictions", what.key)
```